src/snowflake/snowpark/session.py

```diff
@@ -722,7 +736,7 @@ def get_packages(self) -> Dict[str, str]:
     def add_packages(
         self,
         *packages: Union[str, ModuleType, Iterable[Union[str, ModuleType]]],
-        force_push: bool = True,
+        force_push: bool = False,
```

Comment:
I like that idea too.

One reason it could be not ideal is when users want to force_push for certain packages but not for others - which does not seem likely to me.
On the other hand, having it a parameter with session object let's us set one behavior for all package upload activities - this sounds good to me.